app/bundles/EmailBundle/Swiftmailer/Momentum/DTO/TransmissionDTO/ContentDTO/AttachmentDTO.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\EmailBundle\Swiftmailer\Momentum\DTO\TransmissionDTO\ContentDTO;
+
+final class AttachmentDTO implements \JsonSerializable
+{
+    private string $type;
+
+    private string $name;
+
+    private string $content;
+
+    public function __construct(string $type, string $name, string $content)
+    {
+        $this->type    = $type;
+        $this->name    = $name;
+        $this->content = $content;
+    }
```

Comment:
FYI You can use https://www.php.net/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion with Mautic 5:
```suggestion
    public function __construct(private string $type, private string $name, private string $content)
    {
    }
```